### ISSUE TITLE
Haxe editor syntax

### ISSUE THREAD

**Issue by Alexey-T** [cser/temperate#1]
thanks again for the blog post. I see you use Haxe. CudaText has such a lexer, so if you see any issues with it, give the feedback. on https://github.com/Alexey-T/CudaText/issues

the same for C#.
@cser

**Comment by cser** [comment]
> I see you use Haxe

*used in 2014 at last
![image](https://user-images.githubusercontent.com/655339/104312196-567ad080-54e7-11eb-820c-1ab7bc310aa1.png)
Что тут рассказать? Подсветка как подсветка, можно еще попробовать и нестандартные типы подсветить, учитывая что они не могут начинаться с маленькой буквы. На github, кстати, [пытаются подсвечивать](https://github.com/cser/stdjs/blob/master/haxelib/CommonJS.hx), но зачем-то вместе с именами функций оденим цветом. Иногда вообще все, что с большой буквы в Haxe подсвечивают как типы - не сильно страшно получается.

Но это если фанаты CudaText  имеются среди разработчиков на haХе, для кого стараться.

В issues не стал писать, т.к. использовать haXe мне вряд ли ещё придется, получится фидбек "в вакууме" :(

**Comment by Alexey-T** [comment]
спасибо! может и добавлю подсветку слов с бол.буквы.

**Comment by Alexey-T** [comment]
получилось! обновлю через 10 мин.
![Screenshot from 2021-01-12 15-36-29](https://user-images.githubusercontent.com/4154978/104315449-02beb600-54ec-11eb-8a45-c0ac68ba0101.png)

**Comment by cser** [comment]
Да, примерно с такой подсветкой я и работал. При таком подходе, правда,
ВСЕ_КОНСТАНТЫ светятся как типы - но это никого не напрягало

вт, 12 янв. 2021 г. в 15:37, Alexey T. <notifications@github.com>:

**Comment by Alexey-T** [comment]
А вот и нет
![hax](https://user-images.githubusercontent.com/4154978/104316195-29312100-54ed-11eb-8c48-2d2eab6a554a.png)

почему  "haXe"??

https://haxe.org/

**Comment by cser** [comment]
Там конечно, принято писать абривеатуры типа Xml, но в апи и у
пользователей XML встречается. Хотя тут что так что так - не идеально без
семантического разбора, так что выбирайте

>

**Comment by Alexey-T** [comment]
> Там конечно, принято писать абривеатуры типа Xml, но в апи и у пользователей XML встречается. Хотя тут что так что так - не идеально без семантического разбора, так что выбирайте

что выбирать? я свечу regex который начинается с upcase и потом есть хоть одна lowercase letter.

**Comment by cser** [comment]
Да, либо так, но изредка будет XML неподсвеченным и однобуквенные классы в
примерах, либо просто если начинается с большой буквый - но будут светиться
константы. На свой вкус.

вт, 12 янв. 2021 г. в 15:52, Alexey T. <notifications@github.com>:

**Comment by Alexey-T** [comment]
ну ладно, никто не пишет feedback, оставлю как щас.
